### DISCUSSION TITLE
Installed OK (I think) and opens. But fails to open work area.

### DISCUSSION THREAD

**Discussion by johnsantana91709** [givanz/Vvveb#41] [Q&A]
Hi. Sorry for what must be a really simple issue. I've installed Vvveb on my ISP and I can get to the site and log in to the Admin of Vvveb. All is fine. As soon as I click on something (like edit website), I get this error which seems to want to find something locally (on drive D).  It looks like it's looking for a server on my local computer, but the url that I logged on to is on my ISP.The error message (slightly edited for security:

Warning
: file_exists(): open_basedir restriction in effect. File(D:\Inetpub\vhosts\whoitis.com\vvveb.whoitis.com\public\page-cache\vvveb.whoitis.com/admin/?module=tools/cache&action=delete.new) is not within the allowed path(s): (D:/Inetpub/vhosts/whoitis.com\;C:\Windows\Temp\) in
D:\Inetpub\vhosts\whoitis.com\vvveb.whoitis.com\system\page-cache.php
on line
135

The work area the displays: Internal server error!
(error 500)
SQL file D:\Inetpub\vhosts\whoitis.com\vvveb.whoitis.com\plugins\\-plugins\-contact-form\sql\mysqli\-message.sql does not exist for Sql\Plugins\ContactForm\MessageSQL!

Could someone point me in the right direction? Thanks very much. JS

**Comment by givanz** [comment]
Hi

The first issue seems to a be a bug in page cache not detecting admin probably only on windows, you can safely disable page cache from vvveb/.env.php by setting `PAGE_CACHE` to false 
```php 
define('PAGE_CACHE', false);
```

The second issue is a bug that seems to occur only on windows because of different directory separator, I fixed the bug and tested on windows 10 xampp.

To fix this issue please redownload the zip and update the files on your server if it still doesn't work you can send me temporary access ftp/cpanel/ssh using the email from my profile and I will check why is not working.

**Comment by onjaysantana** [comment]
Thank you very much for you kind help. I'm very interested in getting vvVeb up and working and the install seems to go ok, but then I get errors when trying to get to any page including the dashboard. I've tried what I know, including changing the PAGE_CACHE to false. If you could take a look at it and see what's up, I would appreciate it.

You can ftp in at:
Host: 64.20.33.66
Login user name: givanz
Password: vvvebG9+

If I can provide any more info, please let me know. Php on this domain is 8.1. 
Thanks,
John

**Comment by givanz** [comment]
Can you please provide the url or the error messages that are shown on the site? 
Ftp access without website url does not help because the changes can not be tested.

http://64.20.33.66/ shows plesk login

**Comment by onjaysantana** [comment]
Hi, and thanks again for your help. I do feel like a klutz not getting this
to work. I've tried setting it up at a couple of different urls. This one
at least seems to start:
https://affiliates.cinemascore.us/vvveb/
The log in is:  email: ***@***.***   Password: tk7tntis+_).

A screen opens and that might help you (I hope).

There are several things that I'm confused about and maybe understanding
these points could help me do it myself:

* There are at least two versions of vvVeb related packages. One is the
vvVebJS, and the other just vvVeb, which I think is the CMS.
* I've tried both, and I can get to the editor with vvVebjs, but not the
cms.
* With both of the packages there is a subfolder called public with its own
install folder. Don't know it that is correct, or which one is to be used
to install or to use in production.
* I set up separate databases for each of my install locations, so there
should be no conflict.

These are the biggest confusions I have, but again, I sure appreciate your
help.

John

**Comment by givanz** [comment]
Hi

The cms loads from a subfolder on your site, by default it's configured to run from root folder this is why css and images don't load.
Please edit env.php in the root folder and change line 47 to

```php
defined('V_SUBDIR_INSTALL') || define('V_SUBDIR_INSTALL', '/vvveb');
```

https://github.com/givanz/Vvveb/blob/master/env.php#L47

Let me know if it still doesn't work.

The email provided for login is masked by github and login is not possible but if the change to env.php works then it will not be needed.

I will update the documentation and add subfolder installation information, this was not advertised because it's not thoroughly tested, root installation is more common and secure.

---

VvvebJs is a page builder only that can save and load static html files and does not use a database.

Vvveb CMS is a full featured CMS that includes VvvebJs page builder and uses a database to save posts, pages, products etc.

--- 

Please don't forget to change your passwords or edit messages to remove them.
